websites/L/LeXx/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+      "name": "Tim",
+      "id": "277043432063631360"
+    },
+    "url": "lexx.app",
```

Comment:
why did you remove the other urls?

Reply:
cause i noticed that they will be displayed in the store and only the main should be there

Reply:
But now these wont work 
![image](https://user-images.githubusercontent.com/34792838/103253295-def26080-4980-11eb-89ed-6f3497405882.png)

Maybe use regex then?

Reply:
thanks for hint added it to metadata